entry/src/main/config.json

```diff
@@ -0,0 +1,49 @@
+{
+  "app": {
+    "bundleName": "com.example.hassanusman.circulardialogs",
+    "vendor": "example",
+    "version": {
+      "code": 1000000,
+      "name": "1.0.0"
+    },
+    "apiVersion": {
+      "compatible": 5,
+      "target": 5,
+      "releaseType": "Release"
+    }
+  },
+  "deviceConfig": {},
+  "module": {
+    "package": "com.example.hassanusman.circulardialogs",
+    "name": ".MyApplication",
+    "deviceType": [
+      "phone"
```

Comment:
Add other device types in config.json like Tablet and fix issues find with same.

Reply:
Added tablet, tv in commit 5693b42057797e7071ebb39d5c531d4eef3453bd.